### ISSUE TITLE
Consider bigger arrow heads

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#223]
@arouinfar, in a recent interview, a participant with low vision commented that it would be helpful if the arrow heads on the force arrows was larger. For this participant, the arrow heads just looked like a small dot.

While I think adding the zoom and magnification feature to this sim could address this issue, I also wanted to ask if there was any harm in making the arrow heads a little bigger by default? And it might make sense to do it for both the force and the distance arrows.

@amanda-phet, might want to chime in, too.

**Comment by arouinfar** [comment]
Thanks for the feedback @terracoda. I think we can certainly increase the size a bit, though I think we should do it for all ISLC sims.

@zepumph let's increase the size of the arrow head by 30%.

Here's a mockup showing the original arrowhead size (left) vs. the proposed increase (right).
![image](https://user-images.githubusercontent.com/8419308/89591484-840c7100-d807-11ea-8f25-a9d163fe172f.png)

**Comment by terracoda** [comment]
@arouinfar, I agree, for all ISLC sims.

**Comment by terracoda** [comment]
30% increase in size seems like a good number. Let's go with that and see if it helps.

**Comment by jessegreenberg** [comment]
I am working in this sim now so this is something I can do.

**Comment by terracoda** [comment]
@jessegreenberg, are you able to make arrow heads bigger for all the ISLC sims, as @arouinfar noted?

**Comment by jessegreenberg** [comment]
Indeed, arrow heads are larger in all ISLC sims after above commit. @terracoda or @arouinfar can you review?

**Comment by jessegreenberg** [comment]
Discussed today - the DistanceArrow should also have a larger head.

**Comment by jessegreenberg** [comment]
Distance arrow heads also 30% larger with above commit. Ready for review again.

**Comment by arouinfar** [comment]
Looks great across the ISLC suite. Thanks @jessegreenberg.

**Comment by terracoda** [comment]
@jessegreenberg, the arrow heads look good to me, too.
I am not sure what the next step is for this issue? 
I see it has the label fixed and awaiting deploy, so I will un-assign myself.

**Comment by arouinfar** [comment]
@terracoda these changes will sit around until the sim is republished off of master. For GFLB, that will probably happen when we republish with Self-Voicing. It will probably be longer for GFL and CL. I created tracking issues in those repos so we remember that this is an unpublished change.
